documentation/examples/prometheus.yml

```diff
@@ -20,10 +20,10 @@ rule_files:
 # Here it's Prometheus itself.
 scrape_configs:
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
-  - job_name: "prometheus"
+  - job_name: "extended-ceph-exporter"
 
     # metrics_path defaults to '/metrics'
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ["localhost:9090"]
+      - targets: ["localhost:9138"]
```

Comment:
Looks like some other unrelated changes crept into the PR here now?

Reply:
I have removed this unrelated change, PTAL